cernopendata/modules/fixtures/data/records/atlas-2020-1largeRjet1lep.json

```diff
@@ -50,10 +50,15 @@
       ]
     },
     "usage": {
-      "description": "You can access these data through the ATLAS Virtual Machine and Analysis framework. More documentation is available at:",
+      "description": "This dataset is provided by the ATLAS Collaboration <strong>only for Educational purposes and is not suited for scientific publications</strong>.\n You can access these data through the ATLAS Virtual Machine and Software.\n More documentation is available at:",
```

Comment:
Hi @artfisica, FYI I have rebased your PR against `qa`, since this is how we shall be merging for production.

One suggestion: if it is OK with you, I'll replace "Educational" with "educational" in the middle of sentences, such as in the screenshot below, just for grammar purposes. I guess the text being in bold is clear enough:

![art](https://user-images.githubusercontent.com/517546/114512778-c2311e80-9c39-11eb-93da-381c2759bc98.png)